boards/sodaq-autonomo/include/periph_conf.h

```diff
@@ -52,7 +52,7 @@ static const uart_conf_t uart_config[] = {
         .rx_pad   = UART_PAD_RX_1,
         .tx_pad   = UART_PAD_TX_2,
         .flags    = UART_FLAG_NONE,
-        .gclk_src = GCLK_CLKCTRL_GEN_GCLK0
+        .gclk_src = 0
```

Comment:
I prefer a comma, even on the last initializer

Reply:
I added commas everywhere.